### PR TITLE
[Turbo] fix sca

### PR DESCRIPTION
**Pull request by kbond**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Issues        | n/a
| License       | MIT

Not sure how to fix the failing test.

### CODE REVIEW

File: src/Turbo/phpstan.neon.dist [hunk 1]
```diff
@@ -35,3 +35,13 @@ parameters:
 			message: "#^Parameter \\$stimulus of method Symfony\\\\UX\\\\Turbo\\\\Bridge\\\\Mercure\\\\TurboStreamListenRenderer\\:\\:__construct\\(\\) has invalid type Symfony\\\\WebpackEncoreBundle\\\\Twig\\\\StimulusTwigExtension\\.$#"
 			count: 1
 			path: src/Bridge/Mercure/TurboStreamListenRenderer.php
+
+		-
+			message: "#^Call to an undefined method Doctrine\\\\ORM\\\\Event\\\\OnFlushEventArgs\\:\\:getEntityManager\\(\\)\\.$#"
```
Comment (kbond) [inline src/Turbo/phpstan.neon.dist:40]: I thought phpstan understood `method_exists()` 🤷🏻

File: src/Turbo/src/Doctrine/BroadcastListener.php [hunk 2]
```diff
@@ -136,7 +136,7 @@ private function storeEntitiesToPublish(EntityManagerInterface $em, object $enti
                 foreach ($options as $option) {
                     $this->broadcastedClasses[$class][] = $option->newInstance()->options;
                 }
-            } elseif ($this->annotationReader && $options = $this->annotationReader->getClassAnnotations($r ?? new \ReflectionClass($class))) {
+            } elseif ($this->annotationReader && $options = $this->annotationReader->getClassAnnotations(new \ReflectionClass($class))) {
```
Comment (dunglas) [inline src/Turbo/src/Doctrine/BroadcastListener.php]: Why this change?

Comment (kbond) [reply src/Turbo/src/Doctrine/BroadcastListener.php]: `$r` is never null here, it's always set in the above if statement.

Comment (dunglas) [reply src/Turbo/src/Doctrine/BroadcastListener.php]: So we should only keep `$r`, isn't it?

Comment (kbond) [reply src/Turbo/src/Doctrine/BroadcastListener.php]: Heh, yes, you're right

**Comment by smnandre** [comment]
> Not sure how to fix the failing test.

Related to the deprecation / session, no ?

**Comment by kbond** [comment]
> Related to the deprecation / session, no ?

I don't think so, it's a timeout error in `Symfony\UX\Turbo\Tests\BroadcastTest::testBroadcastWithProxy`
